### PR TITLE
chore: add events for sequencer rewards

### PR DESCRIPTION
**Pull request by mtsitrin**
# PR Standards

## Opening a pull request should be able to meet the following requirements

---

For Author:

- [ ]  Targeted PR against correct branch
- [ ]  included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ]  Linked to Github issue with discussion and accepted design
- [ ]  Targets only one github issue
- [ ]  Wrote unit and integration tests
- [ ]  All CI checks have passed
- [ ]  Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code)

---

For Reviewer:

- [ ]  confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ]  Reviewers assigned
- [ ]  confirmed all author checklist items have been addressed

---

After reviewer approval:

- [ ]  In case targets main branch, PR should be squashed and merged.
- [ ]  In case PR targets a release branch, PR should be rebased.

### CODE REVIEW

**Comment by omritoptix** [comment]
@mtsitrin linter fails

File: x/dist/keeper/proposal_handler.go [hunk 1]
```diff
@@ -12,9 +14,9 @@ func HandleCommunityPoolSpendProposal(ctx sdk.Context, k Keeper, p *types.Commun
 		return addrErr
 	}
 
-	// if k.bankKeeper.BlockedAddr(recipient) {
-	// 	return errorsmod.Wrapf(sdkerrors.ErrUnauthorized, "%s is not allowed to receive external funds", p.Recipient)
-	// }
+	if k.bankKeeper.BlockedAddr(recipient) {
```
Comment (mtsitrin) [inline x/dist/keeper/proposal_handler.go:17]: not sure why it was removed before.
it's a check the community spend proposal can't be to blocked addr

File: x/dist/keeper/proposal_handler.go [hunk 2]
```diff
@@ -12,9 +14,9 @@ func HandleCommunityPoolSpendProposal(ctx sdk.Context, k Keeper, p *types.Commun
...
+		return errorsmod.Wrapf(sdkerrors.ErrUnauthorized, "%s is not allowed to receive external funds", p.Recipient)
+	}
```
Comment (danwt) [inline x/dist/keeper/proposal_handler.go:19]:
```suggestion
		return errorsmod.Wrapf(sdkerrors.ErrUnauthorized, "not allowed to receive external funds: recipient: %s", p.Recipient)

	}
```

File: x/hub-genesis/keeper/msg_server.go [hunk 3]
```diff
@@ -3,6 +3,8 @@ package keeper
 import (
 	"context"
 
+	errorsmod "cosmossdk.io/errors"
+
```
Comment (danwt) [inline x/hub-genesis/keeper/msg_server.go:7]: just curious, what's this? why the change?
I dont' recall this from when I worked on cosmos over 1 year ago 😄

**Review by danwt — APPROVED** [review]

File: x/hub-genesis/keeper/msg_server.go (hunk 3 above)
Comment (danwt) [reply x/hub-genesis/keeper/msg_server.go:7]: ah I see the deprecation

**Review by omritoptix — APPROVED** [review]
